### ISSUE TITLE
My antivirus detects it as a trojan and malware

### ISSUE THREAD

**Issue by Sketchulu** [BuilderDolphin/dolphSol-Macro#55]
When i downloaded this program on opera, opera gx blocked it and my antivirus told me it was malware and i dont know what to believe im just worried about my pc because i have alot of data on here. If someone could tell me why that would help a ton thanks!

**Comment by Amraki** [comment]
**TLDR** - This macro is completely fine. The code contains no malware.

AutoHotkey itself is not malware. However, anyone can use AHK to create malicious scripts. This puts AHK as a whole under suspicion. The "better safe than sorry" approach.

Here is a good video on the issue from an AHK-focused YouTube channel:
https://www.youtube.com/watch?v=r_3sN83S2cc

Google returns many results related to AHK false positives:
* https://www.autohotkey.com/boards/viewtopic.php?t=118631
* https://www.autohotkey.com/boards/viewtopic.php?t=90504
* https://www.autohotkey.com/boards/viewtopic.php?t=114763

As a general rule, only download and run files from trusted sources. Discord and GitHub don't inspect files for malicious code as far as I know, but they have communities you can get feedback from.

Thankfully, you already seem to follow that or you likely wouldn't have created this issue. 👍

I hope that helps. Good luck in Sol's!

**Comment by Sketchulu** [comment]
Thanks man, you were a life saver!
